PF/Assignment 1/assign_1.java

```diff
@@ -0,0 +1,178 @@
+package shopping;
+
+import java.util.*;
+
```

Comment:
Add proper doc comment before class name which should describe what you class does and author name like this
/**
 * Item Class represents each item in shopping cart
 * 
 * @author Nikita Sharda
 */